OCAPP/Controllers/Conferences.py

```diff
@@ -100,48 +100,81 @@ def register_user(conference_id):
 			for message in field:
 				flash(message,field)
 		return redirect('/')
-	
 	if request.form['pay'] == 'check_PO':
 		return render_template('confirmation.html')
 	if request.form['pay'] == 'credit_debit':
-
-		member = {
+		member_data = {
 			'id': member.id,
 			'first_name': member.first_name,
 			'last_name': member.last_name,
 			'email': member.email
 			}
-		return render_template('credit_card.html', member=member, conf_id=conf.id)
+		#FIGURING OUT HOW MUCH TO CHARGE TO CREDIT CARD TRANSACTION
+		conf = Conference.get_by_id(conf.id)
+		days = request.form['regis_len']
+		if days == 'friday' or days == 'saturday':
+			day_divisor = 2
+		else:
+			day_divisor = 1
+		member_type = member.type   
+		if member_type == "Professional":
+			member_type_cost = conf.prof_cost
+		elif member_type == "Vendor":
+			member_type_cost = conf.vend_cost
+		else:
+			member_type_cost = conf.stud_cost
+
+		member_cost = member_type_cost/day_divisor
+		return render_template('credit_card.html', member=member_data, conf_id=conf.id, member_cost = member_cost)
 	#send data by calling functions from imported files and sending it the request.form by using request.form.copy()
 
+
+
+# TEST ROUTE FOR RENDERING CREDIT CARD INFORMATION CHETAN 12/20/16
+@app.route('/creditcard')
+def cctest():
```

Comment:
would like to keep it in there temporarily in case we need to do any other testing in the short term. can remove it as soon as we're sure the cc functionality is fully working. helps with debugging as we don't have to register a user every time